app/frontend/src/components/ScoreBar.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Container, LinearProgress, makeStyles } from "@material-ui/core";
+import TextBody from "./TextBody";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    position: "relative",
+    height: "1.6rem",
+    marginInlineStart: 0,
+    marginBottom: theme.spacing(1),
+    width: "100%",
+    maxWidth: 300,
+  },
+  scoreBar: {
+    position: "absolute",
+    width: "100%",
+    height: "100%",
+    borderRadius: ".8rem",
+  },
+  scoreBarLabel: {
+    position: "absolute",
+    width: "100%",
+    lineHeight: "1.6rem",
```

Comment:
```suggestion
    lineHeight: "1.6",
```
I think it's better practice to set this to a unitless value generally for better accessibility. More detailed reason mentioned here - https://css-tricks.com/almanac/properties/l/line-height/

Reply:
I think I have to keep it the same as `height` in the root style, because this is set so the label will be vertically aligned in the score bar.